tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -213,13 +213,16 @@ When('the user uploads a created file {string} using the webUI', function (eleme
   return client.page.personalPage().uploadFile(uploadPath)
 })
 
-When('the user uploads a created file {string} with overwrite using the webUI', function (element) {
-  const uploadPath = path.join(client.globals.mountedUploadDir, element)
-  client.page
-    .personalPage()
-    .selectFileForUpload(uploadPath)
-    .then(() => client.page.personalPage().confirmFileOverwrite())
-})
+When(
+  'the user uploads a created file {string} with overwrite using the webUI',
+  async function (element) {
```

Comment:
if we are not using **await** inside the function then I don't think we need **async** also